MomentoTest/MomentoSignerTest.cs

```diff
@@ -1,99 +1,76 @@
 ﻿using System;
-using Xunit;
+using System.Text;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Logging;
```

Comment:
Don't think we need`Microsoft.IdentityModel.Logging`

Reply:
Ah sorry I was using this to debug 🙇‍♀️ Will remove it.